autoload/sj/c.vim

```diff
@@ -16,6 +16,19 @@ function! sj#c#SplitFuncall()
   let items = sj#ParseJsonObjectBody(start, end)
   let body = '('.join(items, ",\n").')'
 
+  let body = "("
```

Comment:
Since you're overriding the `body` here, could you remove line 17 where it's initially set?

Reply:
Forget to remove the line. Already done in the latest commit